demo/search.html

```diff
@@ -122,7 +122,7 @@
     buttonIndex.addEventListener('click', function () {
         let batchIndex = batchNumber.value;
         search.readJson(batchIndex).then(function (res) {
-            search.indexBatch(res).then(function () {
+            search.indexBatch(JSON.stringify(res)).then(function () {
```

Comment:
@keerthi16 I hope you have considered performance overheads for this. Otherwise, it may get very expensive.

https://github.com/svaarala/duktape/issues/204

Reply:
@VikasShashidhar Just tested the performance of the JSON.stringify() for 
 - 2906469 Lines of JSON content
 - Array length 16800

#### Took - 555.256ms (0.5s)